docs/tutorials/google/start.ipynb

```diff
@@ -1,478 +1,532 @@
 {
```

Comment:
<p>First, decide with which Google Cloud project you will use the Quantum Research Platform. All of your quantum programs and results will live under a project which you specify when creating and running these programs using Quantum Engine. You can use an existing project or create a new project.&nbsp;<a href="https://cloud.google.com/docs/overview/" target="_blank">Learn more about creating a project</a>.  --&gt; First, decide whether you want to <a href="https://cloud.google.com/docs/overview/" target="_blank">create a new Google Cloud Platform project</a>, or use an existing one.</p><p> </p><p>Remove bullet that says "Log in and agree to&nbsp;<a href="https://cloud.google.com/terms/" target="_blank">Terms of Service</a>." It's implied.</p><p> </p><p>Remove "You can run the code by either clicking the play button (pointed by arrow below) or by selecting the block and pressing CTRL-ENTER." </p><p> </p><p>Remove "After you authenticate and allow access to your project, you will be given a string which you should enter into the text box that appears in the run area (and then press return)." -- this is already implied in "follow the authentication flow"</p><p> </p><p>Replace "If this fails, make sure that you have cut and paste the string correctly (e.g. the clipboard button seems to not work for some browser/OS combinations)." and replace to "Contact your partner in the Quantum Hardware Residency Program if you encounter issues at this stage."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>